package.json

```diff
@@ -9,11 +9,11 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">= 5.4.1",
+    "node": "10",
```

Comment:
Shouldn't this be `>= 10`? This won't allow Node 11.

Reply:
NodeJS 11 went [EOL](https://nodejs.org/en/about/releases/) a number of months ago. It's not supported or recommended. It can cause bugs and other issues with a number of libraries.